src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EELocalSymbol.cs

```diff
@@ -79,7 +79,7 @@ public override string Name
 
         internal override SyntaxToken IdentifierToken
         {
-            get { throw ExceptionUtilities.Unreachable; }
+            get { return default; }
```

Comment:
>get { return default; } [](start = 12, length = 23)

Given the changes in ```ExtendBinderChain```, do we still need this change? Let's revert it otherwise.

Reply:
Great! We do not need this change anymore. Reverting. Thanks!

---
In reply to: [161291140](https://github.com/dotnet/roslyn/pull/24198#discussion_r161291140) [](ancestors = 161291140)